### ISSUE TITLE
Add HiPS clone functionality

### ISSUE THREAD

**Issue by cdeil** [hipspy/hips#122]
I think it would be nice to add some functionality and / or documentation how to clone HiPS.

This came up in  https://github.com/gammapy/gamma-sky/issues/65#issuecomment-397299066 where @tboch mentioned that `hipsgen` has a `-clone` option.

We could just solve this via a docs page, explaining how to get a list of files that should be cloned and then do it from Python. E.g. one might be interested only in certain orders or tile formats or regions of the sky. That's already possible but it needs to be explained to users that aren't familiar with the HiPS folders / files. Also, I'm not sure if there are index files about which tiles are available on the server, or if one has to try to fetch all and see if they are there.

We could also add a function or class and / or command line tool that has a few common options what to clone. If we do this, using `click` is very simple and nice. That's what we use in Gammapy and I documented how it work here: http://docs.gammapy.org/dev/scripts/index.html#implementation

@tboch - Thoughts? Is this a useful addition for this Python package, or does `hipsgen` cover this well, and all that's needed is to add some docs there? If we add something here: would you suggest to add some code or just docs?

**Comment by tboch** [comment]
@cdeil : I agree it would be a useful addition. My preference would go to adding some code.

If the Moc.fits file is present, that gives you a good overview of the tiles you need to retrieve.
